content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,57 @@
+ 
+ - **Zone** - This is a drop-down field with all the zones that can be selected. The following tooltip is displayed: "Geographical region.".
+ 
+ - **Does it need Zip/Postal code?** - Toggle switch for either "yes" or "no". By default is set to "yes".
+
+ - **Zip/Postal code format** - This is a mandatory field.
+The following helptext below is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."
+
+ - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
+Under the helptext there are 5 fields that can be pressed, and when pressed on any of the selections will be added to the address format input box:
+     - **Customer** -  in this section it's possible to press the "+" sign to add to the format the following inputs: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
+     - **Warehouse** - in this section it's possible to press the "+" sign to add to the format the following inputs: reference, name, management_type, htmlFields
+     - **Country** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, zip_code_format, htmlFields
+     - **State** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, htmlFields 
+     - **Address** - in this section it's possible to press the "+" sign to add to the format the following inputs: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
+     
+ Below there are 4 buttons with different actions:
+ 
+ 
+     - **Use the last registered format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the last registered formats.
+     - **Use the default format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the default address formats.
+     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
```

Comment:
I don't see any modal on my side when I click on this button

Reply:
@marionf 
![image](https://user-images.githubusercontent.com/75423142/152989498-be06f809-b98c-42bb-98df-ba37b4000f03.png)